library/src/main/java/com/novoda/downloadmanager/lib/DownloadBatch.java

```diff
@@ -2,18 +2,24 @@
 
 import java.util.List;
 
-class DownloadBatch {
+public class DownloadBatch {
```

Comment:
in terms of library writing, we have now made this class available to clients, this means it's implementation is tied in to the version of the library.

I would have done a conversion from this internal represenation to something else `Collated..` and then THAT class can be responsible for keeping public API compatibility and this class can carry on being ruthlessly refactored.

I'd X this if it hadn't already been merged